ocs_ci/utility/utils.py

```diff
@@ -2267,13 +2270,15 @@ def get_az_count():
         return 1
 
 
-def ceph_health_check(namespace=None, tries=20, delay=30):
+def ceph_health_check(namespace=None, tries=20, delay=30, cluster_type="internal"):
```

Comment:
this breaks the existing flow. 
eg: ceph_health_check(tries=30, delay=60)

Reply:
The function example you gave should work. What is the reason for it not to work?